src/EnergyPlus/WeatherManager.cc

```diff
@@ -7976,63 +8012,71 @@ namespace WeatherManager {
+                                        cAlphaFieldNames(3) + '.');
+                        ShowContinueError("...Entered name=\"" + cAlphaArgs(3) + "\".");
+                        ShowContinueError(
+                                "...Should be a full year schedule (\"Schedule:Year\", \"Schedule:Compact\", \"Schedule:File\", or "
+                                "\"Schedule:Constant\" objects.");
+                        ErrorsFound = true;
+                    } else {
                         WPSkyTemperature(Item).IsSchedule = true;
                         WPSkyTemperature(Item).SchedulePtr = Found;
                     }
+                } else { // See if it's a valid schedule.
+                    Found = GetDayScheduleIndex(cAlphaArgs(3));
+                    if (Found == 0) {
+                        ShowSevereError(RoutineName + cCurrentModuleObject + "=\"" + cAlphaArgs(1) + "\", invalid " +
+                                        cAlphaFieldNames(3) + '.');
+                        ShowContinueError("...Entered name=\"" + cAlphaArgs(3) + "\".");
+                        ShowContinueError(
+                                "...Should be a single day schedule (\"Schedule:Day:Hourly\", \"Schedule:Day:Interval\", or \"Schedule:Day:List\" objects.");
+                        ErrorsFound = true;
+                    } else {
+                        if (envFound != 0) {
+                            schPtr = FindNumberInList(Found, SPSiteScheduleNamePtr, NumSPSiteScheduleNamePtrs);
+                            if (schPtr == 0) {
+                                ++NumSPSiteScheduleNamePtrs;
+                                SPSiteScheduleNamePtr(NumSPSiteScheduleNamePtrs) = Found;
+                                SPSiteScheduleUnits(NumSPSiteScheduleNamePtrs) = units;
+                                SetupOutputVariable("Sizing Period Site Sky Temperature Schedule Value", unitType,
+                                                    SPSiteSkyTemperatureScheduleValue(envFound), "Zone", "Average",
+                                                    cAlphaArgs(3));
+                            } else if (SPSiteScheduleUnits(schPtr) != units) {
+                                ++NumSPSiteScheduleNamePtrs;
+                                SPSiteScheduleNamePtr(NumSPSiteScheduleNamePtrs) = Found;
+                                SPSiteScheduleUnits(NumSPSiteScheduleNamePtrs) = units;
+                                SetupOutputVariable("Sizing Period Site Sky Temperature Schedule Value", unitType,
+                                                    SPSiteSkyTemperatureScheduleValue(envFound), "Zone", "Average",
+                                                    cAlphaArgs(3));
+                            }
+                            WPSkyTemperature(Item).IsSchedule = true;
+                            WPSkyTemperature(Item).SchedulePtr = Found;
+                        }
+                    }
```

Comment:
I'm not finding any code that reports the selected sky temperature model in the eio output. That *should* have happenend when WeatherProperty:* was added. Since this can be set per environment, it could be added to this eio output:
```
! <Environment>,Environment Name,Environment Type, Start Date, End Date, Start DayOfWeek, Duration {#days}, Source:Start DayOfWeek,  Use Daylight Saving, Use Holidays, Apply Weekend Holiday Rule,  Use Rain Values, Use Snow Values
```

Reply:
Done.